src/components/input/Input.vue

```diff
@@ -109,9 +109,7 @@
         },
         data() {
             return {
-                // This is necessary because, for some reason, textarea
-                // returns undefined if there's no v-model or value.
```

Comment:
@rafaelpimpa I can't reproduce this problem
```
// This is necessary because, for some reason, textarea
// returns undefined if there's no v-model or value.
```
I've tested textarea type and it seems to be working fine.